starters/remix/app/components/drupal/node/NodeArticle.tsx

```diff
@@ -1,7 +1,7 @@
 import { FragmentOf, readFragment } from 'gql.tada'
 import { Article } from '~/components/ui/Article'
 import { NodeArticleFragment } from '~/graphql/drupal/fragments/node'
-import { extractImageFromMedia, extractUser } from '~/graphql/drupal/helpers'
+import { parseMediaImage, parseUser } from '~/graphql/drupal/helpers'
```

Comment:
The names `parseUser` and `parseMediaImage` are descriptive and indicate that these functions parse user and media image data, respectively. However, if you want to make the names even clearer, you could consider:

- `transformUserData` instead of `parseUser`
- `transformMediaImage` instead of `parseMediaImage`

These names suggest that the functions are transforming or processing the data rather than simply parsing it.

Reply:
I will try `resolveMediaImage` `resolveLink` and `resolveUser` instead what do you thing about?